### ISSUE TITLE
[BugZilla] atmos_awips failed for missing ensemble member in ensstat files

### ISSUE THREAD

**Issue by XianwuXue-NOAA** [NOAA-EMC/GEFS#78]
To fix BugZilla Issue: [#1336](http://www2.spa.ncep.noaa.gov/bugzilla/show_bug.cgi?id=1336)

he current configuration for atmos_awips job fails with degraded geavg* files from atmos_ensstat job. In the parm file for awips job, it is hardcoded to run with data from all 30 members. However, if one member is behind and atmos_ensstat proceeds with less data, awips job fails. (The ensstat job will fail if more than 1 member is missing.)

One possible workaround is to make awips run degraded with less data as in ensstat. However, awips data is important to the users. We would like to deliver the complete products if possible.

Another solution is to let ensstat wait until all the members to complete, so the awips job can always have the full set of data. But in extreme cases, such awips data might get delayed for hours waiting for the slow member.

In short, we have two options. Both of them require minor scripting changes.
1) Ensstat proceeds with 29 members if 1 is behind. AWIPS might be degraded but more likely on time.
**2) Ensstat waits for all 30 members, and AWIPS has all 30 members. No degradation but might be delayed for (up to) a few hours**. 

We can go with either depending on the users' need.

**Comment by XianwuXue-NOAA** [comment]
Fixed in gefs.v12.3.3
